packages/init/index.ts

```diff
@@ -21,3 +22,29 @@ export default function initializeInquirer(...args: string[]): Function | void {
 	}
 	return npmPackagesExists(packages);
 }
+
+export function UIManager(dependencies: string[]) {
+	// tslint:disable-next-line:no-var-requires
+	const npm = require("npm");
```

Comment:
Why not importing at the top of the file?

Reply:
Only required for UI 🙄

Reply:
TS handles that, to only import it when the functions runs?

Also, what about 🙄? Not everybody knows everything about everything. I asked a legitime question.

Reply:
I am an beginner here btw 😀

Reply:
There is a `import()` in TS that dynamic import module.
Should I use that @montogeek . Standard import statement is static btw

Reply:
`import()` was not supported for TS configurations.